backend/src/main/java/com/capstone/moneytree/service/impl/DefaultUserService.java

```diff
@@ -167,7 +172,11 @@ public User updateUser(User userToUpdate, User user) {
         }
 
         if (user.getPassword() != null) {
-            userToUpdate.setPassword(user.getPassword());
+            userToUpdate.setPassword(encryptData(user.getPassword()));
+        }
+
+        if (user.getBiography() != null) {
```

Comment:
This was added and needs a test for this path. Need to keep the coverage high :P